### DISCUSSION TITLE
screen lock stay on at startup

### DISCUSSION THREAD

**Discussion by Typhus14** [joshuar/go-hass-agent#291] [General]
Hello,
screen lock sensor seem to be on at startup ever the session is open. If I lock my screen and unlock it the sensor turn off as expected.
Thank you for your help.

**Comment by joshuar** [comment]
Hey I know why this is. I've been a lazy coder :frowning_face:  I don't check the screen lock state at start up of the agent. I will fix this in the next version. Thank you for reporting!

**Comment by Typhus14** [comment]
Good job anyway for a lazy coder ;-) . Thank you for your answer . I will wait the next version !

**Comment by joshuar** [comment]
Hey there, just to report back, this is fixed with https://github.com/joshuar/go-hass-agent/commit/40cbb57c322da2f7d6b0d2f6c99363158c6cf284 and will be in the next version, to be released shortly...
